playwright/e2e/widgets-basic.spec.ts

```
@@ -11,76 +11,76 @@ test.describe('Landing page widgets - basic presence and links', () => {
     const landing = new LandingPage(page);
-    await expect(landing.widget('rhel-widget')).toBeVisible();
+    await expect(landing.widget('landing-./RhelWidget-widget')).toBeVisible();
   });
 
   test('RHEL widget link targets Insights', async ({ page }) => {
     const landing = new LandingPage(page);
-    await expect(landing.widget('rhel-widget').locator('a')).toHaveAttribute(
+    await expect(landing.widget('landing-./RhelWidget-widget').locator('a')).toHaveAttribute(
       'href',
       /\/insights\//,
     );
```

ESLint

[error] Replace `landing.widget('landing-./RhelWidget-widget').locator('a')).toHaveAttribute(⏎······'href',⏎······/\/insights\//,⏎····` with `⏎······landing.widget('landing-./RhelWidget-widget').locator('a'),⏎····).toHaveAttribute('href',·/\/insights\//`

(prettier/prettier)